libs/vkd3d/command.c

```diff
@@ -6764,24 +6846,24 @@ static bool d3d12_command_list_update_rendering_info(struct d3d12_command_list *
         if (list->dsv.view->format->vk_aspect_mask & VK_IMAGE_ASPECT_DEPTH_BIT)
         {
             if (list->dsv_plane_optimal_mask & (VKD3D_DEPTH_PLANE_OPTIMAL | VKD3D_DEPTH_STENCIL_PLANE_GENERAL))
-            {
-                d3d12_command_list_fuse_attachment_clear(list, &rendering_info->depth,
-                    list->dsv.resource, list->dsv.view, VK_IMAGE_ASPECT_DEPTH_BIT, &barrier);
-            }
+                dsv_optimal_aspects |= VK_IMAGE_ASPECT_DEPTH_BIT;
 
             rendering_info->info.pDepthAttachment = &rendering_info->depth;
         }
 
         if (list->dsv.view->format->vk_aspect_mask & VK_IMAGE_ASPECT_STENCIL_BIT)
         {
             if (list->dsv_plane_optimal_mask & (VKD3D_STENCIL_PLANE_OPTIMAL | VKD3D_DEPTH_STENCIL_PLANE_GENERAL))
-            {
-                d3d12_command_list_fuse_attachment_clear(list, &rendering_info->stencil,
-                    list->dsv.resource, list->dsv.view, VK_IMAGE_ASPECT_STENCIL_BIT, &barrier);
-            }
+                dsv_optimal_aspects |= VK_IMAGE_ASPECT_STENCIL_BIT;
 
             rendering_info->info.pStencilAttachment = &rendering_info->stencil;
         }
+
+        if (dsv_optimal_aspects)
```

Comment:
Does this mean we cannot fuse clears for read-only DSVs? Is that a necessary restriction? iirc the only thing we use read-only DSV for is dynamic depth/stencil test.